gateway/rpc_storage_handler.go

```diff
@@ -978,7 +989,9 @@ func (r *RPCStorageHandler) ProcessKeySpaceChanges(keys []string, orgId string)
 	for _, certId := range CertificatesToRemove {
 		log.Debugf("Removing certificate: %v", certId)
 		r.Gw.CertificateManager.Delete(certId, orgId)
+		r.Gw.RPCCertCache.Delete("cert-raw-" + certId)
```

Comment:
It looks like fix of another bug.

Reply:
(but its fine to keep it here, since now you can have big cache time and issue becomes visible)